stores/user/index.ts

```diff
@@ -1,104 +1,106 @@
-import { UserSession, AppConfig } from 'blockstack'
-import Cookies from 'js-cookie'
+import { UserSession, AppConfig } from 'blockstack';
+import Cookies from 'js-cookie';
 
-const host = typeof document === 'undefined' ? 'https://app.co' : document.location.origin
-const appConfig = new AppConfig(['store_write'], host)
-export const userSession = new UserSession({ appConfig })
+const host =
+  typeof document === 'undefined' ? 'https://app.co' : document.location.origin;
+const appConfig = new AppConfig(['store_write'], host);
+export const userSession = new UserSession({ appConfig });
 
 const initialState = {
   userId: null,
   jwt: null,
   signingIn: false,
   user: null
-}
+};
 
 const constants = {
   SIGNING_IN: 'SIGNING_IN',
   SIGNED_IN: 'SIGNED_IN',
   SIGN_IN: 'SIGN_IN',
   SIGNING_OUT: 'SIGNING_OUT'
-}
+};
 
 const signingIn = () => ({
   type: constants.SIGNING_IN
-})
+});
 
-const signedIn = (data) => ({
+const signedIn = (data: any) => ({
   type: constants.SIGNED_IN,
   payload: {
     token: data.token,
     user: data.user,
     userId: data.user.id
   }
-})
+});
 
 const signOut = () => {
-  Cookies.remove('jwt')
+  Cookies.remove('jwt');
   return {
     type: constants.SIGNING_OUT
-  }
-}
+  };
+};
 
-const handleSignIn = (apiServer) => async (dispatch) => {
-  const token = userSession.getAuthResponseToken()
+const handleSignIn = (apiServer: any) => async (dispatch: any) => {
```

Comment:
`apiServer` is obviously `string`, and you can get `import { Dispatch } from 'redux'` for the `dispatch` type.